.github/workflows/ci-common-steps.yml

```
@@ -0,0 +1,84 @@
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: 10.2.0
+          run_install: false
+
+      - name: Install Node.js ${{ matrix.node-version }}
```

actionlint (1.7.4)

property "node-version" is not defined in object type {}

(expression)